macOS/DuckDuckGo/NavigationBar/View/AddressBarButtonsViewController.swift

```diff
@@ -1525,6 +1525,7 @@ final class AddressBarButtonsViewController: NSViewController {
         button.backgroundColor = .buttonMouseDown
         button.mouseOverColor = .buttonMouseDown
         (popover.contentViewController as? PermissionAuthorizationViewController)?.query = query
+        (popover.contentViewController as? PopupBlockedViewController)?.query = query
```

Comment:
since the popover is typed here, could we update this to have a property defined in the popover subclass?